### ISSUE TITLE
About result on Lyft dataset

### ISSUE THREAD

**Issue by linrui26** [valeoai/PointBeV#8]
Thank you for your work!
Is the vehicle segmentation result on Lyft dataset obtained using temporal settings?

**Comment by LoickCh** [comment]
Hi,
No, without specification the model used is the static one.

**Comment by linrui26** [comment]
Thanks! Can you provide me the config file on Lyft dataset?

**Comment by linrui26** [comment]
And how many samples have training set and validation set respectively in the lyft dataset? I have download the Lyft L5 from https://woven.toyota/en/perception-dataset/, and split data following Fiery, getting training samples 16506 and val samples 6174. Is that right?

**Comment by LoickCh** [comment]
Hi,
For the lyft dataset, you just have to overwrite: 

'''
is_lyft: False
dataroot: ${paths.data_dir}/nuScenes
'''

In the data configuration file

**Comment by linrui26** [comment]
OK. Thank you for your reply!

**Comment by linrui26** [comment]
Hello, has data augmentation been applied to lyft dataset? If so, what should I overwrite？

**Comment by LoickCh** [comment]
Hi,
Yes, the same data augmentation as for nuScenes.
